README.md

```diff
@@ -1,22 +1,24 @@
 yorg
 =========
 
-**Yorg** is an *open source* racing game developed by Ya2 using [Panda3D](http://www.panda3d.org) for *Windows*, *OSX* and *Linux*. More info on [its page](http://www.ya2.it/pages/yorg.html).
+**Yorg** is an *open source* racing game developed by Ya2 using [Panda3D](http://www.panda3d.org) for *Windows*, *OSX* and *Linux*. More informations can be found on [this page](http://www.ya2.it/pages/yorg.html).
```

Comment:
`information` is an uncountable noun in this use, so the use of the plural is not correct.